Samples.Swift/Samples.Swift/SceneDelegate.swift

```diff
@@ -12,41 +11,38 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
   var window: UIWindow?
 
-  @available(iOS 13.0, *)
+
   func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-    // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
-    // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
-    // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-    guard let _ = (scene as? UIWindowScene) else { return }
+    guard let windowScene = (scene as? UIWindowScene) else { return }
+      window = UIWindow(frame: UIScreen.main.bounds)
+      let viewController = CheckoutViewController()
+      window?.rootViewController = viewController
+      window?.makeKeyAndVisible()
+      window?.windowScene = windowScene
```

Comment:
I just got Haider's comment on the ObjC PR and I think it also applies here.

Maybe we can keep `@available(iOS 13.0, *)` here.
Then I think we can move setting rootViewController to `Main.storyboard` like how it was before :smile:

Reply:
Done, I don't think we also need to move to use the `Main.storyboard` for this to work as it was not using it previously here?

Reply:
Updated this in my AppDelegate, we don't need Main.storyboard, just to reference the window and the root controller in AppDelegate for targets less than iOS 13. 😄

Reply:
awesome!!!! yep that works too!!!!! :smile: